src/VirtualList/VirtualList.js

```diff
@@ -7,22 +7,36 @@
 import DOMUtil from '../Util/DOMUtil';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import throttle from 'lodash.throttle';
+
+import BindMixin from '../Mixin/BindMixin';
 import Util from '../Util/Util';
 
 let mathMax = Math.max;
 let mathMin = Math.min;
 let mathFloor = Math.floor;
 let mathCeil = Math.ceil;
 
-class VirtualList extends React.Component {
+class VirtualList extends Util.mixin(BindMixin) {
```

Comment:
Do we really need the `BindMixin`? Would love to slowly reduce the usage of mixins...